test/Test/Toml/Parsing/Unit/Toml.hs

```diff
@@ -8,109 +9,120 @@ import Data.Text (Text)
 
 import Test.Tasty.Hspec (Spec, describe, it)
 
-import Toml.Edsl (mkToml, table, tableArray, (=:))
-import Toml.Type (AnyValue (..), TOML (..), Value (..))
+import Toml.Edsl (empty, mkToml, table, tableArray, (=:))
+import Toml.PrefixTree (pattern (:||))
+import Toml.Type (TOML (..), Value (..))
 
 import qualified Data.List.NonEmpty as NE
 import qualified Data.Text as T
 
-import Test.Toml.Parsing.Unit.Common (day2, dquote, int, intT, makeKey, parseToml, squote, str,
-                                      strT, tomlFailOn, tomlFromArray, tomlFromKeyVal,
-                                      tomlFromTable)
+import Test.Toml.Parsing.Unit.Common (day2, parseToml, tomlFailOn)
 
 tomlSpecs :: Spec
 tomlSpecs = do
     describe "Key/values" $ do
         it "can parse key/value pairs" $ do
-            parseToml "x='abcdef'" (tomlFromKeyVal [(makeKey ["x"], AnyValue (Text "abcdef"))])
-            parseToml "x=1"        (tomlFromKeyVal [(makeKey ["x"], AnyValue (Integer 1))])
-            parseToml "x=5.2"      (tomlFromKeyVal [(makeKey ["x"], AnyValue (Double 5.2))])
-            parseToml "x=true"     (tomlFromKeyVal [(makeKey ["x"], AnyValue (Bool True))])
-            parseToml "x=[1, 2, 3]" (tomlFromKeyVal [(makeKey ["x"] , AnyValue (Array [Integer 1, Integer 2, Integer 3]))])
-            parseToml "x = 1920-12-10" (tomlFromKeyVal [(makeKey ["x"], AnyValue (Day day2))])
+            parseToml "x = 'abcdef'" $ mkToml ("x" =: Text "abcdef")
+            parseToml "x = 1"  $ mkToml ("x" =: Integer 1)
+            parseToml "x = 5.2" $ mkToml ("x" =: Double 5.2)
+            parseToml "x = true" $ mkToml ("x" =: Bool True)
```

Comment:
Let's also remove spaces around `=` in some tests. We also want to test that this parses correctly in different situations:

```haskell
x=true
x= true
x =true
x = true
```

I'm not sure whether we have tests for all these things, but we can just have different space placement in some of the existing tests.